### ISSUE TITLE
support for canonical tags

### ISSUE THREAD

**Issue by markoddi** [fulls1z3/ngx-meta#44]
Hi,

How do I make the canonical tag change. I've got the title, description and keywords changing ok.

Do I simply do this:

```
 {
    path: 'home',
    component: HomeComponent,
    data: {
      meta: {
        title: 'Home page',
        description: 'Description of the home page', 
        canonical: 'mywebsite.com/home' 
      }
    }
  }
```
I'm using angular version 4, with webpack, typescript and ng2-meta plugin. I just need to make the canonical tag change in runtime like the description changes. Is this possible?

**Comment by fulls1z3** [comment]
Hi @markoddi, I've found some useful info on this topic (https://github.com/angular/angular/issues/15776#issuecomment-291862953).

I am planning to update this repo with this feature, on v0.2.0 final release.

**Comment by jesspagan** [comment]
Hi @fulls1z3, I am using v0.4. Is this version going to get an update about this like v.0.2.0??

**Comment by fulls1z3** [comment]
@jesspagan this feature will be implemented both on v0.2.0 and v0.4.0

**Comment by rafa-suagu** [comment]
@fulls1z3 have you some news about this? If you want I can do a PR with this feature

**Comment by fulls1z3** [comment]
@rafa-as sorry for the delay, had busy times. feel free to fire a PR and i'll approve.

**Comment by d-koppenhagen** [comment]
Probably related to: https://github.com/angular/angular/issues/34605

**Comment by rafa-suagu** [comment]
@d-koppenhagen are you working on a Link Service to do a PR to Angular project?

**Comment by d-koppenhagen** [comment]
yes. The PR is already there: https://github.com/angular/angular/pull/34606
